src/main/java/com/ning/http/client/AsyncHttpClientConfig.java

```diff
@@ -1062,7 +1079,8 @@ public Thread newThread(Runnable r) {
                     sslSessionCacheSize, //
                     sslSessionTimeout, //
                     providerConfig, //
-                    maxRequestHeaders);
+                    maxRequestHeaders,
+                    properProxyAuthorization);
```

Comment:
Remove this. Only system property as a fail safe mechanism in AuthenticationUtils.